### DISCUSSION TITLE
SMTP connector as alternative to SES

### DISCUSSION THREAD

**Discussion by whiteagle3k** [juspay/hyperswitch#3241] [Ideas / Feature Requests]
It would be nice to use any SMTP server as alternative to the implemented SES integration, as AWS SES is a bit tricky to get approved to jump to production. It is frustrating to see the whole thing set up and running and being unable to start it just because you can't send emails.

**Comment by manojradhakrishnan** [comment]
Thanks for the suggestion.

Are you using the Hyperswitch among internal users (your known organization email IDs) or for external users with signup and verify?

@jagan-jaya @JeevaRamu0104

**Reply by whiteagle3k** [reply]
known users and organizations

**Reply by manojradhakrishnan** [reply]
Thanks @whiteagle3k 
I can see there is a slack thread running for the solution. Any implementation updates will be posted and tracked here.
